RadarSDK/Radar.m

```diff
@@ -49,7 +49,9 @@ + (void)initializeWithPublishableKey:(NSString *)publishableKey {
 
     [RadarSettings setPublishableKey:publishableKey];
     [[RadarLocationManager sharedInstance] updateTrackingFromInitialize];
-    [[RadarAPIClient sharedInstance] getConfig];
+    [[RadarAPIClient sharedInstance] getConfig:^(RadarStatus unused, RadarMeta *_Nullable meta) {
+        [[RadarLocationManager sharedInstance] updateTrackingFromMeta:meta];
```

Comment:
hmm, yes, so per the discussion above, the caller handles updating the tracking from meta @phil-brown @jrtibbetts

Reply:
I removed the `if` statement at `RadarAPIClient.m`, line 115, and left this and the one below it as-is.